packages/react-core/src/components/Touchspin/examples/Touchspin.md

```diff
@@ -0,0 +1,345 @@
+---
+id: Touchspin
+section: components
+cssPrefix: pf-c-touchspin
+propComponents: ['Touchspin']
+beta: true
+---
+
+## Examples
+
+### Default
+
+```js
+import React from 'react';
+import { Touchspin } from '@patternfly/react-core';
+
+class BasicTouchspin extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: 90
+    };
+
+    this.onMinus = () => {
+      this.setState({
+        value: this.state.value - 1
+      });
+    };
+
+    this.onChange = event => {
+      this.setState({
+        value: Number(event.target.value)
+      });
+    };
+
+    this.onPlus = () => {
+      this.setState({
+        value: this.state.value + 1
+      });
+    };
+  }
+
+  render() {
+    const { value } = this.state;
+    return (
+      <Touchspin
+        value={value}
+        onMinus={this.onMinus}
+        onChange={this.onChange}
+        onPlus={this.onPlus}
+        inputProps={{ name: 'basic-touchspin-input', 'aria-label': 'touchspin input' }}
+        minusButtonProps={{ 'aria-label': 'minus' }}
+        plusButtonProps={{ 'aria-label': 'plus' }}
+      />
+    );
+  }
+}
+```
+
+### With unit
+
+```js
+import React from 'react';
+import { Touchspin } from '@patternfly/react-core';
+
+class UnitTouchspin extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value1: 90,
+      value2: (1.0).toFixed(2)
+    };
+
+    this.onMinus1 = () => {
+      this.setState({
+        value1: this.state.value1 - 1
+      });
+    };
+
+    this.onChange1 = event => {
+      this.setState({
+        value1: Number(event.target.value)
+      });
+    };
+
+    this.onPlus1 = () => {
+      this.setState({
+        value1: this.state.value1 + 1
+      });
+    };
+
+    this.onMinus2 = () => {
+      this.setState({
+        value2: (Number(this.state.value2) - 0.01).toFixed(2)
+      });
+    };
+
+    this.onChange2 = event => {
+      console.log(event.target.value);
+      this.setState({
+        value2: event.target.value
+      });
+    };
+
+    this.onPlus2 = () => {
+      this.setState({
+        value2: (Number(this.state.value2) + 0.01).toFixed(2)
+      });
+    };
+  }
+
+  render() {
+    const { value1, value2 } = this.state;
+    return (
+      <React.Fragment>
+        <Touchspin
+          value={value1}
+          onMinus={this.onMinus1}
+          onPlus={this.onPlus1}
+          inputProps={{ onChange: this.onChange1, name: 'touchspin-input1', 'aria-label': 'touchspin input 1' }}
```

Comment:
Shouldn't you be using the onChange prop here?

Reply:
You can use either because they to the same place, but I'll edit the example for consistency.